.github/workflows/ruby.yml

```diff
@@ -23,8 +23,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['2.7', '3.0', '3.1', '3.2']
-        activerecord: ['6.0', '6.1', '7.0', 'main']
+        ruby-version: ['3.0', '3.1']
```

Comment:
Any reason why you want to remove ruby 3.2 from the tests?
https://ruby-lang.org/en/downloads/branches/
3.1 expected EOL is 2025-03-31 which is next year, if we can check support for 3.2 first for free, why not?

Reply:
actually no reason in particular, other than that our current objective is only upgrading up to Ruby 3.1, and there might be some potential issue should there be modification done from Ruby 3.1 to 3.2 that makes some of the codes here failing

However, just now I tried adding back 3.2, it's still okay so I retrieve it back for testing